app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  def update_counter
```

Comment:
[OPTIONAL]

- You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` like I mentioned in the Post model.

Reply:
good suggestion, I'll apply it in my code